src/libs/utils/hostosinfo.h

```diff
@@ -31,7 +31,7 @@
 
 #include <QString>
 
-#ifdef Q_OS_WIN
+#if defined(Q_OS_WIN) || defined(Q_OS_OS2)
```

Comment:
Qt for OS/2 has a special define for that — Q_OS_DOSLIKE. It's defined both on Windows and on OS/2:
```suggestion
#if defined(Q_OS_DOSLIKE)
```